gfreshii-backend/src/main/java/com/edbootcamp/service/IngredientServiceImpl.java

```diff
@@ -13,36 +13,36 @@
 public class IngredientServiceImpl implements IngredientService{
 
 	@Autowired
-	private IngredientDAO iDao;
+	private IngredientDAO dao;
```

Comment:
I would still refactor this to "ingredientDao", because it is common to have Services with many daos.